npm/src/compiler/worker-browser.ts

```diff
@@ -2,12 +2,20 @@
 // Licensed under the MIT License.
 
 import * as wasm from "../../lib/web/qsc_wasm.js";
-import { log } from "../log.js";
+import { TelemetryEvent, log } from "../log.js";
 import { Compiler } from "./compiler.js";
 import { createCompilerDispatcher } from "./worker-proxy.js";
 
 let invokeCompiler: ReturnType<typeof createCompilerDispatcher> | null = null;
 
+function telemetryHandler(telemetry: TelemetryEvent) {
```

Comment:
Now we have different worker scripts for Node & browser, then each for compiler and language service, this now needs to be in four places. Just leaving in compiler for now until we refactor to simplify.